src/annotator/test/adder-test.js

```diff
@@ -154,6 +154,15 @@ describe('Adder', () => {
       assert.called(adderCallbacks.onShowAnnotations);
     });
 
+    it('hides the adder when `Escape` shortcut is pressed', () => {
+      adder.annotationsForSelection = ['ann1'];
+      showAdder();
+      triggerShortcut('Escape');
```

Comment:
We're not terribly consistent about this in some test suites, but as a rule of thumb once tests exceed 4 or 5 lines I find it helps readability to separate the setup/action being tested/assertions (aka Arrange/Act/Assert), assuming there is a clear separation between these phases. In this case, that means adding new blank lines before and after the `triggerShortcut` call.

Reply:
roger that